### PR TITLE
DOCSUP-5917: CREATE QUOTA and ALTER QUOTA syntax updated

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #11670 (part 3 CREATE|ALTER QUOTA)

### CODE REVIEW

File: docs/en/sql-reference/statements/alter/quota.md [hunk 1]
```diff
@@ -5,16 +5,33 @@ toc_title: QUOTA
...
     [RENAME TO new_name]
-    [KEYED BY {'none' | 'user name' | 'ip address' | 'client key' | 'client key or user name' | 'client key or ip address'}]
+    [KEYED BY {NONE | USER_NAME | IP_ADDRESS | CLIENT_KEY | CLIENT_KEY, USER_NAME | CLIENT_KEY, IP_ADDRESS} | NOT KEYED]
     [FOR [RANDOMIZED] INTERVAL number {SECOND | MINUTE | HOUR | DAY | WEEK | MONTH | QUARTER | YEAR}
-        {MAX { {QUERIES | ERRORS | RESULT ROWS | RESULT BYTES | READ ROWS | READ BYTES | EXECUTION TIME} = number } [,...] |
+        {MAX { {QUERIES | ERRORS | RESULT_ROWS | RESULT_BYTES | READ_ROWS | READ_BYTES | EXECUTION_TIME} = number } [,...] |
         NO LIMITS | TRACKING ONLY} [,...]]
     [TO {role [,...] | ALL | ALL EXCEPT role [,...]}]
 ```
+Multiword key types may be written either with underscores (`CLIENT_KEY`), or with spaces and in simple quotes (`'client key'`). You may also use `'client key or user name'` instead of `CLIENT_KEY, USER_NAME`, and `'client key or ip address'` instead of `CLIENT_KEY, IP_ADDRESS`.
```
Comment (vitlibar) [inline docs/en/sql-reference/statements/alter/quota.md:21]: There were changes in the syntax, and the old syntax is still supported in the code, but I think the documentation should describe only an actual way of writing those quotas. 

Here is the actual way:

```
CREATE QUOTA q12_01297 KEYED BY client_key, ip_address
CREATE QUOTA q6_01297 FOR INTERVAL 2 month MAX queries = 100, errors = 11, result_rows = 1000, result_bytes = 10000, read_rows = 1001, read_bytes = 10001, execution_time = 2.5
```

Comment (vitlibar) [reply docs/en/sql-reference/statements/alter/quota.md:21]: `KEYED BY none` is also an example of the old syntax. It should be written as `NOT KEYED` now.

Comment (olgarev) [reply docs/en/sql-reference/statements/alter/quota.md]: > There were changes in the syntax, and the old syntax is still supported in the code, but I think the documentation should describe only an actual way of writing those quotas.
> 
> Here is the actual way:
> 
> ```
> CREATE QUOTA q12_01297 KEYED BY client_key, ip_address
> CREATE QUOTA q6_01297 FOR INTERVAL 2 month MAX queries = 100, errors = 11, result_rows = 1000, result_bytes = 10000, read_rows = 1001, read_bytes = 10001, execution_time = 2.5
> ```

@vitlibar The idea of including both old and new syntax was to assure the users that their old queries are still okay. If it's not needed, I deleted it.

File: docs/en/sql-reference/statements/create/quota.md [hunk 2]
```diff
@@ -11,19 +11,25 @@ Syntax:
 
 ``` sql
 CREATE QUOTA [IF NOT EXISTS | OR REPLACE] name [ON CLUSTER cluster_name]
-    [KEYED BY {'none' | 'user name' | 'ip address' | 'forwarded ip address' | 'client key' | 'client key or user name' | 'client key or ip address'}]
+    [KEYED BY {USER_NAME | IP_ADDRESS | CLIENT_KEY | CLIENT_KEY, USER_NAME | CLIENT_KEY, IP_ADDRESS} | NOT KEYED]
```
Comment (vitlibar) [inline docs/en/sql-reference/statements/create/quota.md:14]: The command `SHOW CREATE QUOTA` displays the parameters `USER_NAME`, ... and `RESULT_BYTES`, ... in lower case. The command `CREATE QUOTA` can be written with those parameters in upper case, but lower case looks a bit better.

File: docs/en/sql-reference/statements/create/quota.md [hunk 3]
```diff
@@ -11,19 +11,25 @@ Syntax:
...
     [FOR [RANDOMIZED] INTERVAL number {SECOND | MINUTE | HOUR | DAY | WEEK | MONTH | QUARTER | YEAR}
-        {MAX { {QUERIES | ERRORS | RESULT ROWS | RESULT BYTES | READ ROWS | READ BYTES | EXECUTION TIME} = number } [,...] |
+        {MAX { {QUERIES | ERRORS | RESULT_ROWS | RESULT_BYTES | READ_ROWS | READ_BYTES | EXECUTION_TIME} = number } [,...] |
          NO LIMITS | TRACKING ONLY} [,...]]
     [TO {role [,...] | ALL | ALL EXCEPT role [,...]}]
```
Comment (vitlibar) [inline docs/en/sql-reference/statements/create/quota.md:18]: Have we described the key types `USER_NAME`, `IP_ADDRESS`, `CLIENT_KEY` somewhere?

File: docs/en/sql-reference/statements/create/quota.md (hunk 2 above)
Comment (olgarev) [reply docs/en/sql-reference/statements/create/quota.md:14]: @vitlibar We have a guide - all key words must be written in upper case...

File: docs/en/sql-reference/statements/create/quota.md (hunk 3 above)
Comment (olgarev) [reply docs/en/sql-reference/statements/create/quota.md:18]: @vitlibar Updates the syntax, added links. Correct?

Comment (vitlibar) [reply docs/en/sql-reference/statements/create/quota.md:18]: Yes, it looks good.

**Pull request merged**